module/VuFind/tests/unit-tests/src/VuFindTest/I18n/ExtendedIniNormalizerTest.php

```diff
@@ -55,6 +57,23 @@ public function testLanguageFileIntegrity()
         $this->checkDirectory($normalizer, $langDir);
     }
 
+    /**
+     * Test bypassing of non-language-files.
+     *
+     * @return void
+     */
+    public function testLanguageFileCheck()
+    {
+        $file = $this->getFixtureDir() . 'language/base/non-language.ini';
+        $normalizer = new ExtendedIniNormalizer();
+
+        $this->expectExceptionMessage(
```

Comment:
Should we have a second fixture to test the missing equal sign check for completeness?

Reply:
I should have added it when I briefly thought about it, but then forgot. Now added.